x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/constants.ts

```diff
@@ -292,6 +292,48 @@ export const SOURCE_CONFIG_DESCRIPTION = i18n.translate(
   }
 );
 
+export const SYNC_MANAGEMENT_TITLE = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.sync.management.title',
+  {
+    defaultMessage: 'Sync management',
+  }
+);
+
+export const SYNC_MANAGEMENT_DESCRIPTION = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.sync.management.description',
```

Comment:
```suggestion
  'xpack.enterpriseSearch.workplaceSearch.contentSources.syncManagementDescription',
```
We typically use the 4th segment here to namespace in the correct app section. Also prefer to camelCase the last item per the [docs](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md)